lib/msf/core/payload/windows/meterpreter_loader.rb

```diff
@@ -92,8 +97,9 @@ def generate_config(opts={})
   end
 
   def stage_meterpreter(opts={})
+    ds = opts[:datastore] || datastore
     # Exceptions will be thrown by the mixin if there are issues.
-    dll, offset = load_rdi_dll(MetasploitPayloads.meterpreter_path('metsrv', 'x86.dll'))
+    dll, offset = load_rdi_dll(MetasploitPayloads.meterpreter_path('metsrv', 'x86.dll', debug: ds['DEBUG']))
```

Comment:
Thinking aloud; If a developer toggles between building debug builds and production builds locally, they'll have to also remember to update the datastore with the new debug flag. If they don't, it might hard to work out what's gone wrong. I wonder if there's a nice way to streamline that process 🤔